src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class Ui {
+
+    /**
+     * Method that prints to screen a line for ease of reading.
+     */
+    public void line() {
```

Comment:
For consistency you can consider naming this method as printLine() instead. 😄

Reply:
Fixed.